plugins/api-docs/src/components/ApiExplorerPage/DefaultApiExplorerPage.tsx

```diff
@@ -74,6 +102,7 @@ export const DefaultApiExplorerPage = (props: DefaultApiExplorerPageProps) => {
     columns,
     actions,
     ownerPickerMode,
+    filters,
```

Comment:
Could you add a usage example here?
https://github.com/backstage/backstage/blob/master/plugins/api-docs/README.md#customizations